# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/hostap-utils/hotfix-install.patch
# Copyright (C) 2007 - 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- hostap-utils-0.4.7.orig/Makefile
+++ hostap-utils-0.4.7/Makefile
@@ -2,6 +2,8 @@ ifndef CFLAGS
 CFLAGS = -O2 -Wall
 endif
 
+prefix=/usr/local
+
 # Include directory for CVS version
 CFLAGS += -I../driver/modules
 
@@ -28,6 +30,11 @@ hostap_rid: hostap_rid.o util.o
 
 hostap_rid.o: hostap_rid.c util.h
 
+install: all
+	mkdir -p $(DESTDIR)$(prefix)/bin
+	for i in prism2_srec hostap_crypt_conf hostap_diag hostap_io_debug hostap_rid; do \
+		cp $$i $(DESTDIR)$(prefix)/bin/$$i; done
+
 clean:
 	rm -f prism2_srec hostap_crypt_conf hostap_diag hostap_rid \
 		hostap_io_debug *.o
